Directory.Packages.props

```diff
@@ -40,7 +40,7 @@
 
     <!-- Test infra -->
     <PackageVersion Include="Appveyor.TestLogger" Version="2.0.0" />
-    <PackageVersion Include="FluentAssertions" Version="8.2.0" />
+    <PackageVersion Include="FluentAssertions" Version="7.2.0" /> <!-- Do NOT upgrade to v8.x! -->
```

Comment:
Could we try `[7.2.0, 8.0.0)`?

https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges

Reply:
Thé other possibility (for later) : https://github.com/AwesomeAssertions/AwesomeAssertions

Reply:
https://docs.shouldly.org/. This is what I use.

Reply:
> Could we try `[7.2.0, 8.0.0)`?
> 
> https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges

This is a good idea 👍

I've heard Shouldly mentioned in few places, but I don't feel the urgency to migrate off FA right now